flink-tpcds/src/main/java/com/ververica/flink/benchmark/Benchmark.java

```diff
@@ -59,33 +58,54 @@ public class Benchmark {
 	private static final Option PARALLELISM = new Option("p", "parallelism", true,
 			"The parallelism, default is 800.");
 
+    private static final Option MODE =
+            new Option("m", "mode", true, "mode: 'execute' or 'explain'");
+
 	public static void main(String[] args) throws ParseException {
 		Options options = getOptions();
 		DefaultParser parser = new DefaultParser();
 		CommandLine line = parser.parse(options, args, true);
-		run(
-				setUpEnv(
-						requireNonNull(line.getOptionValue(HIVE_CONF.getOpt())),
-						requireNonNull(line.getOptionValue(DATABASE.getOpt())),
-						Integer.parseInt(line.getOptionValue(PARALLELISM.getOpt(), "800"))
-				),
-				getQueries(
-						line.getOptionValue(LOCATION.getOpt()),
-						line.getOptionValue(QUERIES.getOpt())),
-				Integer.parseInt(line.getOptionValue(ITERATIONS.getOpt(), "1"))
-		);
+        System.out.println("args: " + String.join(" ", args));
+
+        String mode = line.getOptionValue(MODE.getOpt(), "execute");
+        TableEnvironment tEnv =
+                setUpEnv(
+                        requireNonNull(line.getOptionValue(HIVE_CONF.getOpt())),
+                        requireNonNull(line.getOptionValue(DATABASE.getOpt())),
+                        Integer.parseInt(line.getOptionValue(PARALLELISM.getOpt(), "800")));
+        LinkedHashMap<String, String> queries =
+                getQueries(
+                        line.getOptionValue(LOCATION.getOpt()),
+                        line.getOptionValue(QUERIES.getOpt()));
+
+        if ("explain".equals(mode)) {
```

Comment:
use `switch`, and throw exception if the mode is unexpected

Reply:
Done!